pipeline/scripts/analysis_pipeline.jl

```diff
@@ -0,0 +1,85 @@
+# Analogy: library(targets) in R
+using DrWatson
+
+# Activate the project environment
+# Analogy: source(functions.R) in targets
+@quickactivate "Analysis pipeline"
+
+# Include the AnalysisPipeline module
+include(srcdir("AnalysisPipeline.jl"));
+# Include some plotting functionality that is useful but not necessary for analysis
+# pipeline
+include(plotsdir("plot_functions.jl"));
+
+@info("""
+      Running the analysis pipeline.
+      ---------------------------------------------
+      Currently active project is: $(projectname())
+      Path of active project: $(projectdir())
+      """)
+
+using .AnalysisPipeline
+
+## Other dependencies
+# Analogy: tar_option_set(...) in targets
+using JLD2, Plots
+
+## Run the pipeline steps
+# Analogy: list(tar_targets...) followed by tar_make(...) in targets
+# This is an intermediate commit that runs but will be updated with a job
+# scheduler for these tasks in the next commit. Probably Dagger.jl
+
+# Default parameter values and plot true Rt
+default_gi_param_dict = default_gi_params()
```

Comment:
could we be really fancy and have a pipeline type and then dispatch on standard function names to make it really easy to overload parts of the pipeline? Then to change the analysis you would just change the target type and have a write your own new methods where you wanted?